.github/workflows/collect.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will install Python dependencies, collect data for states that DON'T use webkit, and run tests on the generated files
```

Comment:
This is fine.  Let's merge these eventually.

Reply:
Agreed, as soon as we can get the webkit workflow working consistently they should be merged.